README.md

```diff
@@ -59,12 +59,12 @@ be utilized by a light client implementation.
 ```
 ├── beefy-gadget  // The BEEFY gadget
 │  └── ...
+├── beefy-node    // A Substrate node running the BEEFY gadget
+│  └──  ...
 ├── beefy-pallet  // The BEEFY pallet.
 │  └──  ...
 ├── docs          // Documentation
 │  └──  ...
-├── node-example  // A Substrate node running the BEEFY gadget
-│  └──  ...
 ├── primitives    // The BEEFY primitives crate includig a test light client
```

Comment:
If we are renaming things, perhaps let's rename `primitives` as well?

Reply:
Done